src/ComputeSharp.D2D1/Shaders/Interop/Effects/TransformMappers/D2D1DrawInfoUpdateContextImpl.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using TerraFX.Interop.DirectX;
+using TerraFX.Interop.Windows;
+#if NET6_0_OR_GREATER
+using RuntimeHelpers = System.Runtime.CompilerServices.RuntimeHelpers;
+#else
+using RuntimeHelpers = ComputeSharp.D2D1.NetStandard.System.Runtime.CompilerServices.RuntimeHelpers;
+#endif
+
+namespace ComputeSharp.D2D1.Shaders.Interop.Effects.TransformMappers;
+
+/// <summary>
+/// An implementation of the <see cref="ID2D1DrawInfoUpdateContext"/> and <see cref="ID2D1DrawInfoUpdateContexInternal"/> interfaces.
+/// </summary>
+internal unsafe partial struct D2D1DrawInfoUpdateContextImpl
+{
+#if !NET6_0_OR_GREATER
+    /// <inheritdoc cref="QueryInterface"/>
+    [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+    private delegate int QueryInterfaceDelegate(D2D1DrawInfoUpdateContextImpl* @this, Guid* riid, void** ppvObject);
+
+    /// <inheritdoc cref="AddRef"/>
+    [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+    private delegate uint AddRefDelegate(D2D1DrawInfoUpdateContextImpl* @this);
+
+    /// <inheritdoc cref="Release"/>
+    [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+    private delegate uint ReleaseDelegate(D2D1DrawInfoUpdateContextImpl* @this);
+#endif
+
+    /// <summary>
+    /// The shared vtable pointer for <see cref="D2D1DrawInfoUpdateContextImpl"/> instance, for <see cref="ID2D1DrawInfoUpdateContext"/>.
+    /// </summary>
+    private static readonly void** VtblForID2D1DrawInfoUpdateContex = InitVtblForID2D1DrawInfoUpdateContexAndID2D1DrawInfoUpdateContexInternal();
```

Comment:
`Contex[t]`

Reply:
okay I'm not tagging any more of these, but you should do a search to find any that are left :)